dev-util/md4c/md4c-0.4.8.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A C Markdown parser"
+DESCRIPTION="MD4C is Markdown parser implementation in C, with the following features:
+* Compliance: Generally, MD4C aims to be compliant to the latest version of CommonMark specification. Currently, we are fully compliant to CommonMark 0.30.
+* Extensions: MD4C supports some commonly requested and accepted extensions.
+* Performance: MD4C is very fast.
+* Compactness: MD4C parser is implemented in one source file and one header file. There are no dependencies other than standard C library.
+* Embedding: MD4C parser is easy to reuse in other projects, its API is very straightforward: There is actually just one function, md_parse().
+* Push model: MD4C parses the complete document and calls few callback functions provided by the application to inform it about a start/end of every block, a start/end of every span, and with any textual contents.
+* Portability: MD4C builds and works on Windows and POSIX-compliant OSes.
+* Encoding: MD4C by default expects UTF-8 encoding of the input document.
+* Permissive license: MD4C is available under the MIT license.
+"
+HOMEPAGE="https://github.com/mity/md4c/"
+COPYRIGHT="2016-2021 Martin Mitáš"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/release-$portVersion.tar.gz"
+CHECKSUM_SHA256="4a457df853425b6bb6e3457aa1d1a13bccec587a04c38c622b1013a0da41439f"
+SOURCE_DIR="md4c-release-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	md4c$secondaryArchSuffix = $portVersion
+	cmd:md2html$secondaryArchSuffix
```

Comment:
Since we now move the binary to commandBinDir there wouldn't be a need for secondaryArchSuffix here, just use commandSuffix which will be empty:
```diff
-	cmd:md2html$secondaryArchSuffix
+	cmd:md2html$commandSuffix
```